specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-11-15/timeseriesinsights.json

```diff
@@ -1139,21 +1176,7 @@
     "ResourceProperties": {
       "properties": {
         "provisioningState": {
```

Comment:
With this change, we lost the "Provisioning state of the resource." in the generated code:
> Provisioning state of the resource. Possible values include: 'Accepted', 'Creating', 'Updating', ...

Can you check to see if adding a description here will add it back? I know the enum value itself has that description, but it doesn't seem to be picked up.